integ/lib/render-struct.ts

```diff
@@ -45,7 +45,7 @@ export class RenderStruct extends Construct {
     }
 
     const host = 'renderqueue';
-    const zoneName = Stack.of(this).stackName + '.local';
+    const zoneName = Stack.of(this).stackName.slice(0, 45) + '.local';
```

Comment:
I know this gave you headaches to root-cause. Can we add a comment here so these efforts are documented? Same applies to the other occurrence in `integ/lib/testing-tier.ts`

Reply:
Added

Reply:
Thanks!